test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -37,7 +37,7 @@ class test_ucp_tag_xfer : public test_ucp_tag {
         modify_config("MAX_RNDV_LANES", "2");
         m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_ENABLE", "y"));
         if (RUNNING_ON_VALGRIND) {
-            m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_MAX_BCOPY", "8k"));
+            m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_SEG_SIZE", "8k"));
```

Comment:
you introduced TM_MAX_SIZE name, not TM_SEG_SIZE

Reply:
fixed the name of the config attribute